Maui.RevenueCat.InAppBilling/Maui.RevenueCat.InAppBilling.csproj

```diff
@@ -54,4 +54,19 @@
       <PackagePath>\</PackagePath>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Platforms\iOS\Extensions\EntitlementInfosExtensions.cs">
+      <ExcludeFromCurrentConfiguration>false</ExcludeFromCurrentConfiguration>
+    </Compile>
+    <Compile Update="Platforms\iOS\Extensions\OwnershipTypeExtensions.cs">
+      <ExcludeFromCurrentConfiguration>false</ExcludeFromCurrentConfiguration>
+    </Compile>
+    <Compile Update="Platforms\iOS\Extensions\PeriodTypeExtensions.cs">
+      <ExcludeFromCurrentConfiguration>false</ExcludeFromCurrentConfiguration>
+    </Compile>
+    <Compile Update="Platforms\iOS\Extensions\StoreExtensions.cs">
+      <ExcludeFromCurrentConfiguration>false</ExcludeFromCurrentConfiguration>
+    </Compile>
+  </ItemGroup>
```

Comment:
what is this for ?

Reply:
Great Question, the IDE added them, but they are useless since they default to false, removed...